spec/spec_helper.rb

```diff
@@ -2,6 +2,9 @@
 
 ENV['RACK_ENV'] = 'test'
 
+require 'bundler'
+Bundler.require :default, :test
```

Comment:
This is required for `defined?(Mongoid::History)` to be true in the integration spec.